packages/cordova/src/ios/AMBInterstitial.swift

```diff
@@ -40,17 +43,26 @@ class AMBInterstitial: AMBAdBase, GADFullScreenContentDelegate {
             ctx.error("Ad is not loaded")
         }
     }
+    
+    func clear() {
+        if (interstitial != nil) {
+            interstitial?.fullScreenContentDelegate = nil
+            interstitial = nil;
+        }
+    }
 
     func adDidRecordImpression(_ ad: GADFullScreenPresentingAd) {
         self.emit(AMBEvents.interstitialImpression)
     }
 
     func ad(_ ad: GADFullScreenPresentingAd, didFailToPresentFullScreenContentWithError error: Error) {
         self.emit(AMBEvents.interstitialShowFail, error)
+        self.clear();
     }
 
     func adDidPresentFullScreenContent(_ ad: GADFullScreenPresentingAd) {
         self.emit(AMBEvents.interstitialShow)
+        self.clear();
```

Comment:
@yigit-serin Would you move this line above `self.emit` so it aligns with Android implement?